katalogus/utils.py

```diff
@@ -1,7 +1,8 @@
 from typing import Type, List
+
+from katalogus.client import get_katalogus, Plugin
 from octopoes.models import OOI
 from tools.models import Organization
-from katalogus.client import get_katalogus, Plugin
 
 
 def get_enabled_boefjes_for_ooi_class(ooi_class: Type[OOI], organization: Organization) -> List[Plugin]:
```

Comment:
This reference to the organization would be more like the rest if it was available on an KAT-alogus client instance, like the octopoes client. Move to future refactoring imho.